recipes/r-spp/meta.yaml

```diff
@@ -39,7 +39,7 @@ requirements:
 test:
   commands:
     - $R -e "library('spp')"
-
+    - $R -e "library('spp'); read.bam.tags(system.file('extdata', 'ex1.bam', package='Rsamtools'))"
```

Comment:
@bgruening can you please advise on how I should quote this for running with involucro? I can't seem to get it right.